inc/media.php

```diff
@@ -1873,6 +1873,10 @@ function media_uploadform($ns, $auth, $fullscreen = false){
     printf($lang['maxuploadsize'],filesize_h(media_getuploadsize()));
     echo '</p>'.NL;
 
+    echo '<div id="allowedmime-accordion">';
```

Comment:
IDs should always have a double underscore to avoid name clashes with auto generated IDs for headlines. However, I would prefer a class over an id here.

Reply:
I substituted the id with a class.